stubs/redis/redis/commands/core.pyi

```diff
@@ -112,10 +112,10 @@ class ManagementCommands:
     def memory_malloc_stats(self, **kwargs): ...
     def memory_usage(self, key, samples: Any | None = ..., **kwargs): ...
     def memory_purge(self, **kwargs): ...
-    def ping(self, **kwargs): ...
+    def ping(self, **kwargs) -> bool: ...
```

Comment:
Does this really return a boolean? Seems like it should return `"PONG"`, because redis-py calls `PING` with no arguments. https://redis.io/commands/ping

Reply:
There's a response callback that compared to the response to `"PONG"`. Same for `save()`.